### DISCUSSION TITLE
Monotonic Everything

### DISCUSSION THREAD

**Discussion by Durahl** [supermerill/SuperSlicer#2174] [Q&A]
Greetings!

I'm currently working on a [Cake Carrier/Protector](https://i.imgur.com/4yod9If.jpg) ( as small as possible Plate on a giant ass bearing assisting in decorating/cutting it plus a 12cm tall Cover with a CNCed Plexi top ) and while thinking about a solution for marking the top of the platter I realized that simply using the Octagram Spiral Pattern would really lend itself as a neat cutting guide for even sized pieces ( plus an inlay for centering ).

Thing is... There is no Monotonic Infill option for that ( and some of the other Infills ) so when doing that pattern on like a circle it'll start with 1 of the outside corners until it hits a threshold and continues with doing the entire star followed by going back to finishing the 7 remaining corners thus creating the unsightly seam where the new infill connects to the previous one:
![IMG](https://i.imgur.com/vq57VE1.jpg)

So, I've been wondering about the possibility of providing this ( or rather all? ) Infills with a monotonic option?

**Comment by supermerill** [comment]
don't need all the 'monotonic region' stuff. It just needs some more constraints on what can be reordered at gcode step. Open an issue for that.

**Comment by AlAmantea** [comment]
Could you not simply add small indicator lines to your CAD file for this? It would be a much faster option in my opinion.
